x-pack/plugins/apm/server/routes/infrastructure/get_host_names.ts

```diff
@@ -23,26 +21,21 @@ interface Aggs extends estypes.AggregationsMultiBucketAggregateBase {
   }>;
 }
 
-interface InfraPlugin {
-  setup: InfraPluginSetup;
-  start: () => Promise<InfraPluginStart>;
-}
-
 const getHostNames = async ({
-  esClient,
+  infraMetricsClient,
   containerIds,
-  index,
   start,
   end,
 }: {
-  esClient: ElasticsearchClient;
+  infraMetricsClient: InfraClient;
   containerIds: string[];
-  index: string;
   start: number;
   end: number;
 }) => {
-  const response = await esClient.search<unknown, { hostNames: Aggs }>({
-    index: [index],
+  const response = await infraMetricsClient.search<
```

Comment:
The type of the response failed because of the type of response from the infraMetricsClient 
<img width="1086" alt="Screenshot 2022-09-26 at 09 31 01" src="https://user-images.githubusercontent.com/31922082/192230461-51b15ecb-a9de-49c3-88ac-67b927bb4187.png">
<img width="1326" alt="Screenshot 2022-09-26 at 09 30 30" src="https://user-images.githubusercontent.com/31922082/192230502-cda0908f-ec22-4a13-98e8-fb1d2a724505.png">

<img width="1326" alt="Screenshot 2022-09-26 at 09 30 44" src="https://user-images.githubusercontent.com/31922082/192230489-57a823cb-6cbe-46c5-81b6-7b4441aeed8f.png">